packages/common/src/tools/timer/NativeTimer.ts

```diff
@@ -0,0 +1,7 @@
+import { Timer } from './types'
+
+export const NativeTimer: Timer = {
```

Comment:
Maybe this should be camelCase since this isn't a class?
```suggestion
export const nativeTimer: Timer = {
```